src/angular-app/bellows/shared/pick-list-editor.module.ts

```diff
@@ -25,14 +25,15 @@ export class PickListEditorController implements angular.IController {
     }
   }
 
-  pickAddItem(): void {
-    if (this.newValue) {
-      const key = PickListEditorController.keyFromValue(this.newValue);
-      this.items.push({ key, value: this.newValue });
-      this.isDeletable[key] = true;
-      this.newValue = undefined;
-    }
-  }
+  //Used in adding an item to the Options lists under Configuration, which is currently disabled in Language Forge 2022-10
+  // pickAddItem(): void {
+  //   if (this.newValue) {
+  //     const key = PickListEditorController.keyFromValue(this.newValue);
+  //     this.items.push({ key, value: this.newValue });
+  //     this.isDeletable[key] = true;
+  //     this.newValue = undefined;
+  //   }
+  // }
```

Comment:
```suggestion
```
Let's just delete it like you did the HTML.  We can revert the commit if we want it back.